src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3782,6 +3782,7 @@ private static void ThrowTypeOrFormatErrors(string resourceString, string errorM
         /// <param name="name"></param>
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'blrv1dr4.wyr'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 170
170:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```